package.json

```diff
@@ -60,6 +60,50 @@
     "test:generate-app": "node test/create-test-app.js",
     "doc:api": "node scripts/open-api/serve.js"
   },
+  "resolutionsInfoForDevelopers": {
```

Comment:
I can remove the resolutions from this P/R cause I already have fixed the locked version.... 

But thinking of it, it's nice to keep it temporarily cause it really says: 

- We had 2 versions of jest (jest-cli 29.5 and jest 29.0) explicitly installed (and their deps)

Here's how to clean our lock... in case someone is working on a jest upgrade to not get in a situation where there's possibility of mismatching deps.

I might remove it if it looks weird.

See also https://github.com/strapi/strapi/pull/16702